addons/account/tests/test_tour.py

```diff
@@ -1,28 +1,21 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import Command
 import odoo.tests
-
+from odoo.tests.common import users
 
 @odoo.tests.tagged('post_install_l10n', 'post_install', '-at_install')
 class TestUi(odoo.tests.HttpCase):
 
+    @users('admin')
     def test_01_account_tour(self):
-        # Reset country and fiscal country, so that fields added by localizations are
-        # hidden and non-required, and don't make the tour crash.
-        # Also remove default taxes from the company and its accounts, to avoid inconsistencies
-        # with empty fiscal country.
-        self.env.company.write({
-            'country_id': None, # Also resets account_fiscal_country_id
-            'account_sale_tax_id': None,
-            'account_purchase_tax_id': None,
-        })
-        account_with_taxes = self.env['account.account'].search([('tax_ids', '!=', False), ('company_id', '=', self.env.company.id)])
-        account_with_taxes.write({
-            'tax_ids': [Command.clear()],
-        })
+        if self.env.company.country_id != self.env.company.account_fiscal_country_id:
```

Comment:
For some l10n modules, some required data are needed to complete
  the tour.
e.g. l10n_mx_edi required some certificates files to be upload and set on the company.
Since (most of the time) the extra data are added to a new company (except for the country_id field that might be changed on the main company but not account_fiscal_country_id), to maximize the chance to have a fully configured company, we chose the one that have the same country as fiscal country.